docs/processors/standalone/building.mdx

```diff
@@ -0,0 +1,141 @@
+---
+title: 'Build your own'
```

Comment:
I was on edge between `Building` and `Build your own`, WDYT? @lovromazgon @raulb

Reply:
@maha-hajja my proposal is (also mainly for your SEO purposes) to go with `How to build your own`.

Reply:
Actually, if we go with the route of "How to", it's quite possible that most of our pages become "how to..." which could turn counterproductive. 

@maha-hajja If we go with how other pages are titled, maybe it should be "Building".

Reply:
I don't have strong opinions here, I think `Build your own` is not bad though 🤷